src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/icons.fluent.scss

```diff
@@ -6452,3 +6452,8 @@
 .bit-icon--StoreLogoMed::before {
     content: '';
 }
+
+.bit-icon--ChevronDown::before {
```

Comment:
isn't this defined before?
or is it a different icon than the one that's already defined?

Reply:
I don't know. I could not find it!

Can you take a look and tell me if there is?

Reply:
what?
a simple search would show the evidence:

![image](https://user-images.githubusercontent.com/6169846/119500545-2b4d9b00-bd7d-11eb-885a-68dd6a56f13b.png)

Reply:
Very strange I did not find it :(  
thanks anyway